spec/factories/workflows.rb

```diff
@@ -57,6 +57,23 @@
     retired_set_member_subjects_count 0
     subject_selection_strategy "builtin"
 
+    strings({
```

Comment:
These strings don't appear to be included in an update to the workflow contents. Other strings are scoped to a particular trait/task type. Are these meant to be the default?

Reply:
Yeah these were the ones that were in the default factory for workflow_content, so it used to have these via the primary content.

Reply:
So do you still need the workflow contents factory of can that file be deleted?